### DISCUSSION TITLE
Do we need `prettier` on `@remix-run/dev`

### DISCUSSION THREAD

**Discussion by fernandocanizo** [remix-run/remix#8474] [Q&A] [answered]
Hi all, I noticed `prettier` is required for `@remix-run/dev`, not under `devDependencies` but as a normal dependency. Possibly there's something I'm missing, but off the top of my head I cannot reason why Prettier would be needed on production for an application that compiles your code and runs it.

So I was thinking it maybe slipped-in somehow and nobody has noticed.

It's not a big deal, but since that way is usable from your project if you are using `npm`, it could lead to problems for the unaware.

Example:
```shell
npx create-remix@latest deleteme-npm
# ...
cd deleteme-npm
npx prettier --version
2.8.8
```
**npm reports prettier version from the package required by`@remix-run/dev`**

```shell
pnpx create-remix deleteme-pnpm
cd deleteme-pnpm
pnpx prettier --version
# ...
3.1.1
```

**pnpm reports the (currently) latest `prettier` version from npm registry**

**Comment by MichaelDeBoey** [comment, marked as answer]
`prettier` is used in `https://github.com/remix-run/remix/blob/main/packages/remix-dev/cli/useJavascript.ts`, which is used by the `reveal` command

https://github.com/remix-run/remix/blob/2ba86f11d36f3fa047a086a2cc8ce3b430afab84/packages/remix-dev/cli/useJavascript.ts#L25-L29
https://github.com/remix-run/remix/blob/2ba86f11d36f3fa047a086a2cc8ce3b430afab84/packages/remix-dev/cli/commands.ts#L285-L291
